core/store/migrations/0033_flux_monitor_round_stats_fk_index.go

```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up33 = `
+CREATE INDEX flux_monitor_round_stats_job_run_id_idx ON flux_monitor_round_stats (job_run_id);
+CREATE INDEX flux_monitor_round_stats_v2_pipeline_run_id_idx ON flux_monitor_round_stats_v2 (pipeline_run_id);
```

Comment:
These are necessary otherwise every row deleted from `job_runs` or `pipeline_runs` creates a sequential scan on `flux_monitor_round_stats` which can have millions of rows.